client/src/components/Container.jsx

```diff
@@ -21,20 +24,32 @@ class Container extends React.Component {
       reviews: [],
       reviewImages: [],
       showModal: false,
-      showMore: false,
+      show: 4,
+      modalReview: null
     };
   }
 
   componentDidMount() {
     this.getData();
   }
 
-  toggleModal() {
+  openModal(reviewImgIndex) {
     this.setState({
+      modalReview: reviewImgIndex,
       showModal: !this.state.showModal
     });
   }
 
+  closeModal(e) {
+    console.log(`e.target.id: ${e.target.id}`);
+    if (e.target.id === 'modal-overlay' || e.target.id === 'close-modal') {
```

Comment:
Does this allow you to close it if you click outside of the modal box?

Reply:
yep! that's what the `modal-overlay` id is. I realized that how i had it previously was closing the modal whenever i clicked on the main modal itself